### PR TITLE
replace: add page

### PR DESCRIPTION
**Pull request by CCreativeCND**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/windows/replace.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# replace
+
+> Replaces files. It is related to robocopy, move, copy, and del. More information: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/replace.
```
Comment (navarroaxel) [inline pages/windows/replace.md:3]:
```suggestion
> Replace files. It is related to robocopy, move, copy, and del commands.
> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/replace>.
```

File: pages/windows/replace.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Replace the destination file with the one from the source directory:
+
+`replace {{path/to/file_or_directory}} {{path/to/destination}}`
+
+- Only adds files that do not already exist:
+
+`replace {{path/to/file_or_directory}} {{path/to/destination}} /a`
+
+- Prompt before it replaces each file:
```
Comment (navarroaxel) [inline pages/windows/replace.md:13]: Maybe this is better:
```suggestion
- Interactively copy multiple files, with a prompt before replacing or adding a destination file:
```

File: pages/windows/replace.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# replace
+
+> Replaces files. It is related to robocopy, move, copy, and del. More information: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/replace.
+
+- Replace the destination file with the one from the source directory:
+
+`replace {{path/to/file_or_directory}} {{path/to/destination}}`
+
+- Only adds files that do not already exist:
```
Comment (navarroaxel) [inline pages/windows/replace.md:9]:
```suggestion
- Add files to the destination directory instead of replacing existing files:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @CCreativeCND, thanks for your submission. I left some comments.

File: pages/windows/replace.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Wait for you to insert a disk before it replaces files (originally to allow inserting a floppy disk):
+
+`replace {{path/to/file_or_directory}} {{path/to/destination}} /w`
+
+- Replace all files in subfolders of the destination:
+
+`replace {{path/to/file_or_directory}} {{path/to/destination}} /s`
+
+- Only replaces files in the destination directory which are older than the files in the source directory:
```
Comment (navarroaxel) [inline pages/windows/replace.md:30]:
```suggestion
- Replace only files in the destination directory which are older than the files in the source directory:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @CCreativeCND :slightly_smiling_face:

**Pull request merged**